fatips/1.md

```diff
@@ -6,221 +6,252 @@
 digital goods.
 
 
 # Specification
 
 
-## Token ID  `extends` [FATIP-100](100.md)
-
+## Token ID  `extends` [FATIP-0](0.md)
 
 ## Addresses `extends` [FATIP-0](0.md)
 
+Instead of crediting and debiting numeric quantities of tokens as in FATIP-0, this standard credits and debits token IDs to and from addresses. As such, address balances are represented as an array of owned token IDs.
 
-## Token Issuance Chain
 
-Unlike FAT-0, the issuance chain can contain multiple entries. FAT-1 allows the
-ongoing issuance of unique tokens over time up to a maximum supply.
 
 
-### Issuance Chain ID Derivation  `extends` [FATIP-100](100.md)
+## Token Chain `extends` [FATIP-0](0.md)
 
 
-### Token Issuance Entry `extends` [FATIP-101](101.md)
+### Token Issuance Entry `extends` [FATIP-0](0.md)
 
 FAT-1 token issuance entries include a additional `tokens` field, which is an
-array of unique token objects, each signed by the issuing identity .
+array of unique individual token objects being issued.
+
+
+
 
-Every token in the issuance must be valid for the issuance entry to be valid.
-Issuing a token does not place the token into the circulating supply. Once
-issued the token can be sent via coinbase transaction by ID to it's initial
-bearer.
+#### Individual FAT-1 Tokens
 
+Each token in the `tokens` field must contain a unique `id` within the scope of this
+issuance. I.e If FAT-1 token type A is created, having two individual tokens issued on it with the `id` `0` is illegal. The second issuance will be invalid.
 
-#### Individual FAT-1 Tokens `extends` [FATIP-101](101.md)
+No content enforcements are placed on `id`, so as to allow the issuer to integrate their FAT-1 issuance into existing business practices if needed. It's best to choose short IDs since doing so maximizes the amount of
+tokens that can be issued in a single issuance, if possible. As with any Factom entry, a maximum of 10KB of FAT-1 token issuance content is possible. Large quantity issuances in excess of that limit should be split into multiple entries.
 
-Each token in the `tokens` field must contain a unique ID with respect to this
-FAT-1 token. I.e having two `greple` tokens with the ID `0` is illegal.
-However, having a `greple` token with ID `0` and a `testfat0token` with ID `0`
-is legal. It's best to choose short IDs since doing so maximizes the amount of
-tokens that can be issued in a single issuance. As with any Factom entry, it's
-maximum size can be 10KB in content.
+Note that issuing a token does not place the token into the circulating supply. Once
+issued the token can be sent via coinbase transaction by `id` to it's initial
+bearer. If `supply` is enforced, the issuer can only issue up to `supply` count of individual tokens. In the edge case of a batch issuance pushing the total supply over `supply`, the cutoff is calculated from the 0th token in the array.
 
+Token issuance entries are ongoing like transaction entries to allow ongoing issuance of tokens by the issuer.
```

Comment:
This sentence made me more confused. Could you rephrase it differently for me to help me understand the information you are trying to convey here?

Reply:
Absolutely. More confused? What was confusing before this so that I/we can fix it? :wink:

Reply:
I just mean I'd be fine of this sentence didn't existed, but after reading it I was really like, what does he mean exactly, am I missing an important point? 
In the end you are just saying that there can be new issuance of new tokens?

Reply:
Saying that there can be new issuance of new tokens. If the issuer submits the initial issuance entry and issues tokens `A`, `B`, `C`, they can then a week later submit another issuance entry to issue tokens `D` & `E`. The parallel drawn with transactions is that since issuance entries can occur multiple times, they require similar security considerations to transactions (which are not currently reflected in the spec).

Reply:
Ok I see. I don't know maybe it is just me but I just found the meaning of this sentence hard to grasp. I'd have gone with something more straightforward first like "It is possible to issue new tokens at any time by creating a new issuance entry" and then if you want to mention the security considerations that stem from that fact I would do it explicitly (rather than implicitly by drawing a parallel with transactions).